commands/Tools/contact.js

```diff
@@ -0,0 +1,37 @@
+exports.run = async (client, msg, [type, ...content]) => {
+  // Determines which type of contact and sends to the appropriate channel.
+  let output = "";
+  if (type === "bug") {
```

Comment:
What about a switch? More performant and cleaner. And don't use `+=` but `=` (you're not appending that string into the string.

Additionally, a shorter and performant solution would be:

```javascript
const types = {
  bug: "Bug Report",
  idea: "New Idea",
  message: "New Message",
};

// ...

return this.channel.send(`${types[type]} ...`);
```

Reply:
Talking with Gus about this, having 3 pathways for the if else and having it in a switch case doesn't make much of a noticeable difference in performance, but if Kyra suggests the switch case, then go ahead.